pullrequest.js

```diff
@@ -77,6 +93,25 @@ chrome.storage.sync.get({url: ''}, function(items) {
 
         $body.on('click', '.js-collapse-deletions', collapseDeletions);
 
+        if (items.tabSwitchingEnabled) {
```

Comment:
No, reloading the page makes sense, that's what I meant by on the fly, I see how it will work now. It's good.